transcom/mymove
# B 20673

## [B-20673](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A982871)

## Summary

Will need new payment request status for TPPS RECEIVED
We send TPPS an invoice, EDI 858 with payment request information
TPPS will receive this information and send back an EDI 997
The receipt of an EDI 997 confirms that the TPPS has received the invoice
(TBD) An 824 will provide more detail in addition to the 997 for what errors, if any, came with the sending 
Based on data we receive about how TPPS pays HomeSafe (receiving the 997 (possibly receiving the 824), we will need to build backend logic for updating the status of the payment request to reflect those updates
UI portion: add the TPPS received status to PaymentRequestCard.jsx




## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app
2. Login as a TIO
3. approve a payment request
4. SINCE files aren't sent to get locally or integration- the next step is to find that approved payment request and update the status to 'TPPS Received' 
5. verify the status updated on the UI

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![Screenshot 2024-07-29 at 11 45 59 AM](https://github.com/user-attachments/assets/5dcbf90e-a528-4397-82e6-1e10dda300e0)
